src/load.ts

```diff
@@ -116,7 +118,7 @@ export default async function load(
     if (urlCache.has(schemaID)) return options.schemas; // exit early if this has already been scanned
     urlCache.add(schemaID); // add URL to cache
 
-    let contents = "";
+    let contents: string;
```

Comment:
minor nitpick, but could this be changed back to `""` for now? it makes runtime more predictable if this initialized to a `string` prototype and has access to all of those methods. an empty string is still falsy